examples/application_commands/cog_example.py

```diff
@@ -0,0 +1,27 @@
+from nextcord import Interaction, message_command, slash_command, user_command
+from nextcord.ext import commands
+
+
+class ApplicationCommandCog(commands.Cog):  # Subclassing commands.Cog as you always do.
+    def __init__(self, bot):  # takes in bot for the setup later
```

Comment:
```suggestion
    def __init__(self, bot: commands.Bot):  # takes in bot for the setup later
```
Typehinting should be added here for consistency.